### ISSUE TITLE
Includes VAT text not possible to translate in language files

### ISSUE THREAD

**Issue by renesejling** [woothemes/woocommerce#10329]
Please see this: https://www.evernote.com/l/AIo6b4Xc5kFJO6wx4IN-qlbkCbkVYXxUm6g

**Comment by mikejolley** [comment]
Please see this: https://wordpress.org/support/topic/postcode-zip-label-not-being-picked-up-for-translation?replies=8

tl;dr the translation on .org needs to be completed and approved.

**Comment by renesejling** [comment]
I’ll check it out, thanks.

Any idea who does the danish translation, a lot is missing when upgrading from below 2.5+


Den 11. feb. 2016 kl. 16.24 skrev Mike Jolley <notifications@github.com<mailto:notifications@github.com>>:


Please see this: https://wordpress.org/support/topic/postcode-zip-label-not-being-picked-up-for-translation?replies=8

tl;dr the translation on .org needs to be completed and approved.

**Comment by renesejling** [comment]
Hello again

Where do I find the admin translation for WooCommerce.
Is that part of this: https://translate.wordpress.org/projects/wp-plugins/woocommerce/

Thanks
René

Den 11. feb. 2016 kl. 16.24 skrev Mike Jolley <notifications@github.com<mailto:notifications@github.com>>:


Please see this: https://wordpress.org/support/topic/postcode-zip-label-not-being-picked-up-for-translation?replies=8

tl;dr the translation on .org needs to be completed and approved.

**Comment by mikejolley** [comment]
@renesejling there is only 1 POT. Thats all .org supports.

**Comment by renesejling** [comment]
Ok so it’s all in that one .pot file. Thanks
I got access to the danish translation on wordpress.org<http://wordpress.org> and will be working on it a bit to finish it also :)

Den 17. feb. 2016 kl. 10.40 skrev Mike Jolley <notifications@github.com<mailto:notifications@github.com>>:


@renesejling<https://github.com/renesejling> there is only 1 POT. Thats all .org supports.
